src/sdk/pynni/tests/test_model_speedup.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import os
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import math
+from unittest import TestCase, main
+
+from nni.graph_utils import build_module_graph
+from nni.compression.torch import L1FilterPruner
+from nni.compression.speedup.torch import ModelSpeedup
+
+class BackboneModel1(nn.Module):
```

Comment:
I think we should add tests for graph utils, as PyTorch is known to have issues on its tensorboard graph. For example, https://github.com/pytorch/pytorch/issues/33691 and https://github.com/pytorch/pytorch/issues/33670

Reply:
Maybe we can "borrow" their tests: https://github.com/pytorch/pytorch/blob/253943d5a7bb90a420e4d94366101915823c7929/test/test_tensorboard.py#L505

Reply:
Good suggestion,  added the borrowed test case from pytorch v1.4.